web/src/js/routes/EmergencyRoute.jsx

```diff
@@ -37,9 +37,6 @@ class EmergencyRoute extends React.Component {
               );
             })}
           </div>
-          <div className='EmergencyRoute-kyr'>
```

Comment:
Would you actually bring this back? It'd be useful to be able to point people to for "how to render markdown directly"